src/components/Board.js

```diff
@@ -89,14 +100,17 @@ class Board extends React.Component {
   }
 
   render() {
-    const { totalData, totalContributions, totalDistance, isLoading } = this.state;
+    const { totalData, totalContributions, totalDistance, isLoading, startsWithSearch } = this.state;
     return (
       <MainContainer>
         <a href="/"><Img src={logo} alt="MapSwipe logo" /></a>
-        <FormContainer>
-          <Input type="text" onBlur={(e) => { this.handleOnBlur(e); }} onKeyUp={(e) => { this.handleKeyUp(e); }} />
-          <SubmitButton onClick={() => { this.runSearch(); }}>Search</SubmitButton>
-        </FormContainer>
+        <SearchBar
+          handleOnBlur={(e) => { this.handleOnBlur(e); }}
```

Comment:
More extraneous anonymous function creation here - these can all be in the form `handleOnBlur={this.handleOnBlur}` (applies to all the other attributes here too)

Reply:
@bencpeters 
Okay got it. Then I will `bind` this to `state` functions